### PR TITLE
Bug 1926021: Update buildah to v1.19.8, containernetworking/cni to v0.8.1

### PR DESCRIPTION
**Pull request by gabemontero**
per the cni cve bug discussion during planning, I was curious

it built and passed unit tests no problem

if it passes e2e's I'll see about applying bz to this, start "formal" review

@adambkaplan 
@nalind - this jumps from 1.16 to 1.19

### CODE REVIEW

**Comment by nalind** [comment]
> per the cni cve bug discussion during planning, I was curiou

I lack the context here, but should this have updated `github.com/containernetworking/cni`?

> @nalind - this jumps from 1.16 to 1.19

That's fine.

File: go.sum [hunk 1]
```diff
@@ -179,26 +181,23 @@ github.com/containerd/typeurl v0.0.0-20180627222232-a93fcdb778cd h1:JNn81o/xG+8N
...
-github.com/containernetworking/cni v0.7.2-0.20190904153231-83439463f784/go.mod h1:LGwApLUm2FpoOfxTDEeq8T9ipbpZ61X79hmU3w8FmsY=
 github.com/containernetworking/cni v0.8.0 h1:BT9lpgGoH4jw3lFC7Odz2prU5ruiYKcgAjMCbgybcKI=
 github.com/containernetworking/cni v0.8.0/go.mod h1:LGwApLUm2FpoOfxTDEeq8T9ipbpZ61X79hmU3w8FmsY=
-github.com/containers/buildah v1.16.4 h1:bxthp2FoGcpc2O/RyvbGUAZoefmc5hRBqWQi3BjRu7w=
-github.com/containers/buildah v1.16.4/go.mod h1:i1XqXgpCROnfcq4oNtfrFEk7UzNDxLJ/PZ+CnPyoIq8=
-github.com/containers/common v0.21.0 h1:v2U9MrGw0vMgefQf0/uJYBsSnengxLbSORYqhCVEBs0=
-github.com/containers/common v0.21.0/go.mod h1:8w8SVwc+P2p1MOnRMbSKNWXt1Iwd2bKFu2LLZx55DTM=
-github.com/containers/image/v5 v5.10.1 h1:tHhGQ8RCMxJfJLD/PEW1qrOKX8nndledW9qz6UiAxns=
-github.com/containers/image/v5 v5.10.1/go.mod h1:JlRLJZv7elVbtHaaaR6Kz8i6G3k2ttj4t7fubwxD9Hs=
+github.com/containernetworking/cni v0.8.1 h1:7zpDnQ3T3s4ucOuJ/ZCLrYBxzkg0AELFfII3Epo9TmI=
```
Comment (gabemontero) [inline go.sum:186]: @adambkaplan @nalind here is the bump to v0.8.1 noted in https://bugzilla.redhat.com/show_bug.cgi?id=1926021#c1

**Comment by gabemontero** [comment]
> > per the cni cve bug discussion during planning, I was curiou
> 
> I lack the context here, but should this have updated `github.com/containernetworking/cni`?

Exactly.  See https://github.com/openshift/builder/pull/229#pullrequestreview-617818237
> 
> > @nalind - this jumps from 1.16 to 1.19
> 
> That's fine.

Perfect.  You correctly inferred the unspoken confirmation from you we were interested in @nalind - thanks :-)

**Comment by gabemontero** [comment]
tests all green

/assign @adambkaplan

**Review by adambkaplan — COMMENTED** [review]
@gabemontero since this is just a vendor bump, you can squash commits.

**Comment by gabemontero** [comment]
> @gabemontero since this is just a vendor bump, you can squash commits.

will do ... just broke them up to illustrate what changed when

**Comment by adambkaplan** [comment]
/retitle Bug 1926021: Update buildah to v1.19.8, containernetworking/cni to v0.8.1

**Comment by gabemontero** [comment]
commits squashed @adambkaplan

**Comment by nalind** [comment]
LGTM

**Comment by gabemontero** [comment]
> LGTM

feel free to do the official `/lgtm` @nalind :-)

**Comment by gabemontero** [comment]
> /lgtm

thanks @nalind 

e2e-aws-builds failure stems from the quay.io kerfuffle mentioned in announce-testplatform

/test e2e-aws-builds

**Pull request merged**